### PR TITLE
LPS-188020 Fix I18N for ES modules

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

**Comment by izaera** [comment]
ci:test:sf

**Comment by izaera** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-importmaps-extender/src/main/java/com/liferay/frontend/js/importmaps/extender/internal/servlet/taglib/JSImportMapsExtenderTopHeadDynamicInclude.java
Comment (bryceosterhaus) [inline modules/apps/frontend-js/frontend-js-importmaps-extender/src/main/java/com/liferay/frontend/js/importmaps/extender/internal/servlet/taglib/JSImportMapsExtenderTopHeadDynamicInclude.java:198]: What happens here exactly? Will it just emit this error and then override the existing scope? Should this be if/else?

I would think in the case where a second scope is added, it shouldn't override the existing one.

File: modules/apps/frontend-js/frontend-js-web/src/main/java/com/liferay/frontend/js/web/internal/js/importmaps/extender/FrontendJsWebJSImportMapsContributor.java
Comment (bryceosterhaus) [inline modules/apps/frontend-js/frontend-js-web/src/main/java/com/liferay/frontend/js/web/internal/js/importmaps/extender/FrontendJsWebJSImportMapsContributor.java:1]: Would it make sense for this file to exist in `portal-language` modules?

**Review by bryceosterhaus — APPROVED** [review]
Looks good. I don't see anything that would be blocking from forwarding, just had two small questions

File: modules/apps/frontend-js/frontend-js-importmaps-extender/src/main/java/com/liferay/frontend/js/importmaps/extender/internal/servlet/taglib/JSImportMapsExtenderTopHeadDynamicInclude.java
Comment (izaera) [reply modules/apps/frontend-js/frontend-js-importmaps-extender/src/main/java/com/liferay/frontend/js/importmaps/extender/internal/servlet/taglib/JSImportMapsExtenderTopHeadDynamicInclude.java:198]: Nope, if the exception is thrown the execution stops. So, once a scope is set it remains forever unless it is uninstalled.

File: modules/apps/frontend-js/frontend-js-web/src/main/java/com/liferay/frontend/js/web/internal/js/importmaps/extender/FrontendJsWebJSImportMapsContributor.java
Comment (izaera) [reply modules/apps/frontend-js/frontend-js-web/src/main/java/com/liferay/frontend/js/web/internal/js/importmaps/extender/FrontendJsWebJSImportMapsContributor.java:1]: Mmm, I would say no because portal-language is a JAR with no code, AFAIK. It's only used as a container for lang keys. But it should be Brian who replies that, I guess...

**Comment by dsanz** [comment]
I'm putting this one on hold as we prefer a simpler fix first (https://github.com/liferay-frontend/liferay-portal/pull/3479), then retake this one once merged and Iván is back

**Comment by izaera** [comment]
Closing this in favor of https://github.com/liferay-frontend/liferay-portal/pull/3479. Will reintroduce it in https://liferay.atlassian.net/browse/LPS-188392.

**Pull request closed without merging**
